### ISSUE TITLE
[Bug] Icons from Toolbaritem are not displayed on UWP if starts on second monitor

### ISSUE THREAD

**Issue by Sonic3R** [xamarin/Xamarin.Forms#7505]
### Description

The icons from ToolbarItem elements are not displayed if application starts on second monitor.

### Steps to Reproduce

1. Create tabbed page project and choose only UWP
2. Add the line inside `<TabbedPage ...` node

```
<TabbedPage.ToolbarItems>
    <ToolbarItem Order="Primary" Text="Tab 1" Icon="{x:OnPlatform iOS='sync.png', UWP='Assets/LargeTile.scale-100.png'}" />
    <ToolbarItem Order="Primary" Text="Tab 2" Icon="{x:OnPlatform iOS='settings.png', UWP='Assets/LargeTile.scale-200.png'}"/>
  </TabbedPage.ToolbarItems>
```

3. Run application to second monitor

Attach project:

https://1drv.ms/u/s!AgtERRLVkXN4hLdai20VKSiboOSTkw?e=nL2fJC

### Expected Behavior
Expect to see icons. If you use navigation page, then icons are displayed back.

### Basic Information

- Version with issue: Xamarin.Forms 3.4.0.1008975 . It happens in 4.2.0.778463 too
- IDE: VS 2017
- Platform Target Frameworks:
  - UWP:  16299 
- Affected Devices: Windows laptop

### Screenshots

![image](https://user-images.githubusercontent.com/6311947/64858527-55563c80-d630-11e9-8700-f8c8f00b889f.png)

![no_icon_toolbaritem](https://user-images.githubusercontent.com/6311947/64858970-96028580-d631-11e9-986d-3138464b79ae.gif)

**Comment by rmarinho** [comment]
I can repo it, but i also had sometimes it worked fine.

![TestAppToolbar](https://user-images.githubusercontent.com/1235097/65261174-8d152680-daff-11e9-879e-7fda1cec7015.JPG)

**Comment by SamuelDebruyn** [comment]
Same with our app. It seems to happen a lot when you switch monitors or change the DPI of one of your monitors, but it's hard to repro. However, it's an important issue as the complete UX of the app suffers from it.

**Comment by dimigithub** [comment]
Same issue. We are considering not putting our UWP into production for our end-users because of this bug.

**Comment by jfversluis** [comment]
@SamuelDebruyn @dimigithub do you have any additional information on this at all? Anything can help :) 

I'm trying to look into this and experimenting with what it could be, but so far nothing.

**Comment by jfversluis** [comment]
Small update on our end; we have confirmed that this is a bug in UWP and not Xamarin.Forms. Unfortunately, it will probably take a while before it's fixed.

In the meantime we might want to look into working around this. Are there any suggestions from you on how to do that? 

As mentioned in my previous comment: while testing this I've uncovered that it works fine whenever we use an image as the content for the `AppBarButton`. However, the look will be a bit different than from how it is now. Would that be a solution for you?

**Comment by dimigithub** [comment]
Hello Gerarld,

Sorry for my late reply.

I've tested with the App5 UWP app, and I can not  reproduce the error, 
Not with `<AppBarButton.Content><Image Source="bank.png" />                </AppBarButton.Content>`
, Not even with `<AppBarButton.Icon><BitmapIcon UriSource="ms-appx:///bank.png"/>                </AppBarButton.Icon>`

I still can reproduce the error with the 'TestAppToolbar.UWP" from @Sonic3R

![image](https://user-images.githubusercontent.com/33088294/67197015-38194880-f3fc-11e9-91dd-f26e7aec008c.png)

So,  yes, the option to work with AppBarButton in a native UWP app is an option. But the problem is that our app is xamarin application. So we have  to converted entirely to a UWP app, which is currently not an option.

**Comment by jfversluis** [comment]
@dimigithub No problem!

Just to be clear: I did not mean that you would have to convert your app to a UWP app and not use Xamarin anymore. I can see how it might have gotten you confused by how I put it, sorry about that!

At this point I'm trying to figure out how we can fix this on the XamForms side without having to wait before this bug is fixed in UWP. One of the possible solutions is to make XamForms use this approach: 

```
<AppBarButton.Content>
    <Image Source="bank.png" />
</AppBarButton.Content>
```

But it will look a little different than when you set it as they icon. Have you noticed that?
So, what I actually meant to ask was: if we were to build a solution that would show the icon as an image, but it will look a bit different than how it looks today, would that be a solution for you? Or should we invest more time in finding a solution what will mimic the exact look and feel as it is today?

**Comment by dimigithub** [comment]
Hi @jfversluis,

Ok, sorry I misunderstood.
Yes, that's definitely an option. The different appearance is not an issue for us.

Thanks.

**Comment by Sonic3R** [comment]
@jfversluis Is a good idea but not for us because most UI are in shared Forms library.

We have like:

`<ToolbarItem Order="Primary" Command="{Binding ABCDECommand}" Text="{Binding [ATitle]}" Icon="{x:OnPlatform iOS='folder.png', UWP='Assets/folder.png'}" />`

**Comment by jfversluis** [comment]
@Sonic3R what is you concern here?

Sorry, I might have been causing some confusion by mixing in UWP controls here. Any solution/workaround will ideally mean that you do not have to change any code in your shared library. Or very minimal at least. Code sharing will still work, no or minimal changes on your project.

The thing I am trying to figure out here is how we can solve this in our Forms codebase. Building the toolbar a bit different so we work around this bug. Changing the "translation" so to say that we do within Forms from the abstract Forms `ToolbarItem` to a UWP `AppBarButton`. 

However, if we make changes there, these changes will impact everyone that is using this functionality today whenever they update to a new Forms version. Because of that, we need to be careful about how we want to approach this.

I hope this makes it a bit more clear.

**Comment by samhouts** [comment]
closed by #8147
